src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -31,8 +31,8 @@ export function staticKeyToShadowsocksSessionConfig(staticKey: string): Shadowso
       password: config.password.data,
       prefix: config.extra['prefix'],
     };
-  } catch (error) {
-    throw new errors.ServerAccessKeyInvalid(error.message || 'Failed to parse static access key.');
+  } catch (cause) {
+    throw new errors.ServerAccessKeyInvalid(cause.message || 'Failed to parse static access key.', {cause});
```

Comment:
Shall we just `throw new errors.ServerAccessKeyInvalid('Failed to parse static access key.', {cause})`? And then we can concatenate all error messages (along the errors chained by `cause`) when we need to display it to the user.